src/library/artist/ArtistDetails.vue

```diff
@@ -132,8 +140,21 @@
       isFavourite(): boolean {
         return !!this.favouriteStore.artists[this.id]
       },
-      albums(): Album[] {
-        return orderBy(this.item?.albums ?? [], 'year', this.mainStore.artistAlbumSortOrder)
+      albums(): { releaseType: string, albums: Album[] }[] {
```

Comment:
could probably be made a bit more efficient and/or extracted out

needs to 
- group flat album list by type
- sort those group by Albums at the top, etc